l5-operator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.16 as builder
```

Comment:
We might as well use `1.17` since that's the latest version and what we are using in other projects